src/ntttcp.c

```diff
@@ -26,9 +26,11 @@ void default_ntttcp_test(struct ntttcp_test *test)
 	test->multi_clients_mode	= false;
 	test->last_client		= false;
 	test->use_epoll			= false;
+	test->source_addr_bind 		= false;
```

Comment:
Let's refactor the code as now we have 2 IP address, one could be server_address; and the new one would be client_address.